laws/src/main/scala/cats/laws/discipline/NonEmptyTraverseTests.scala

```diff
@@ -38,6 +38,7 @@ trait NonEmptyTraverseTests[F[_]] extends TraverseTests[F] with ReducibleTests[F
     EqA: Eq[A],
     EqB: Eq[B],
     EqXYFC: Eq[X[Y[F[C]]]],
+    EqXFA: Eq[X[F[A]]],
```

Comment:
I think this is still failing and binary incompatible.

Reply:
looks unused to me, so probably just a leftover

Reply:
Thank you for pointing it out. It was a leftover, just removed that :+1: